operator/config/grafana/controller-resources-metrics-configmap.yaml

```diff
@@ -2,10 +2,10 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: controller-resources-metrics-dashboard
-  namespace: system
+  namespace: kyma-system
```

Comment:
I think this should be kcp-system

Reply:
or rather "system" as it gets replaced in the kustomization

Reply:
this is required by KCP grafana, it only check configmap in `kyma-system`.